src/internal/auth/cf_auth_middleware_test.go

```diff
@@ -107,7 +107,6 @@ var _ = Describe("CfAuthMiddleware", func() {
 			Expect(tc.spyLogAuthorizer.sourceIDsCalledWith).To(HaveKey(sourceID))
 		},
 			Entry("without slash", "12345"),
-			Entry("with slash", "12/345"),
```

Comment:
Do we ever expect the source ID to include a slash?

Reply:
good question. Not sure.

Reply:
This should be safe.